src/webgl/shaders/lighting.glsl

```diff
@@ -130,7 +141,14 @@ vec3 calculateImageSpecular( vec3 vNormal, vec3 vViewPosition ){
 #endif
   // this is to make the darker sections more dark
   // png and jpg usually flatten the brightness so it is to reverse that
-  return pow(outColor.xyz, vec3(10.0));
+  if(metallic != 0.0){
```

Comment:
Right now there's a bit of a jump between the two branches, since when `metallic` is 0.00001, we return `pow(outColor.xyz, vec3(8.0)`, but when `metallic` is sliiightly smaller at `0.0` exactly, we return `pow(outColor.xyz, vec3(10.0)`. Maybe we can remove the if/else and use `10.0` instead of `8.0`?